tests/app/main/test_suppliers.py

```diff
@@ -16,38 +18,34 @@
     VatNumberForm
 )
 
-find_frameworks_return_value = {
+
+def framework_stub_dates(month):
+    return {
+        'applications_close_at': datetime(2000, month, 3),
+        'intention_to_award_at': datetime(2000, month, 4),
+        'clarifications_close_at': datetime(2000, month, 1),
+        'clarifications_publish_at': datetime(2000, month, 2),
+        'framework_live_at': datetime(2000, month, 5),
+    }
+
+
+FIND_FRAMEWORKS_RETURN_VALUE = {
```

Comment:
Would a global like this normally go above the function definition?

Reply:
I think we tend to define globals at the top of the file. Either way, this isn't a new global that I've added - I just renamed it to all caps so it's more obviously a global.